python/lsst/ts/mtaos/mtaos.py

```diff
@@ -951,11 +950,6 @@ async def _execute_ofc(
             self.log.debug("Calculate the subsystem correction successfully.")
 
             await self.pubEvent_degreeOfFreedom()
-            await self.pubEvent_mirrorStresses()
```

Comment:
I thought we went through this already... The reason we do this here and not when issuing correction is to allow us to inspect the results before we apply them.